src/main/java/Duke.java

```diff
@@ -47,50 +48,52 @@ public static void addTodo(String todo) {
+    public static void addEvent(String event) {
+        try {
+            String description = event.split("/at")[0].strip();
+            String date = event.split("/at")[1].strip();
+            Event t = new Event(description, date);
+            tasks[currentTask] = t;
+            currentTask++;
+            System.out.println(line);
+            System.out.println("\tGot it. I've added this task:");
+            System.out.println("\t   [" + t.getType() + "][" + t.getStatusIcon() + "] "
+                    + t.getDescription() + " " + t.getDate());
+            System.out.println("\tNow you have " + Integer.toString(currentTask) + " tasks in the list.");
     	    System.out.println(line + "\n");
-		}
-		catch (ArrayIndexOutOfBoundsException e) {
-			System.out.println(line);
-			System.out.println("\tPlease specify event date.");
-			System.out.println(line + "\n");
-		}
-	}
+        }
+        catch (ArrayIndexOutOfBoundsException e) {
+            System.out.println(line);
+            System.out.println("\tPlease specify event date.");
+            System.out.println(line + "\n");
+        }
+    }
 
-	public static void addDeadline(String deadline) {
-		try {
-			String description = deadline.split("/by")[0].strip();
-			String date = deadline.split("/by")[1].strip();
-			Deadline t = new Deadline(description, date);
-			tasks[currentTask] = t;
-			currentTask++;
-			System.out.println(line);
-	        System.out.println("\tGot it. I've added this task:");
-			System.out.println("\t   [" + t.getType() + "][" + t.getStatusIcon() + "] " + t.getDescription() + " " + t.getDate());
-			System.out.println("\tNow you have " + Integer.toString(currentTask) + " tasks in the list.");
-	        System.out.println(line + "\n");
-		}
-		catch (ArrayIndexOutOfBoundsException e) {
-			System.out.println(line);
-			System.out.println("\tPlease specify deadline.");
-			System.out.println(line + "\n");
-		}
-	}
+    public static void addDeadline(String deadline) {
+        try {
+            String description = deadline.split("/by")[0].strip();
+            String date = deadline.split("/by")[1].strip();
+            Deadline t = new Deadline(description, date);
+            tasks[currentTask] = t;
+            currentTask++;
+            System.out.println(line);
+            System.out.println("\tGot it. I've added this task:");
+            System.out.println("\t   [" + t.getType() + "][" + t.getStatusIcon() + "] "
+                    + t.getDescription() + " " + t.getDate());
+            System.out.println("\tNow you have " + Integer.toString(currentTask) + " tasks in the list.");
+            System.out.println(line + "\n");
+        }
+        catch (ArrayIndexOutOfBoundsException e) {
+            System.out.println(line);
+            System.out.println("\tPlease specify deadline.");
+            System.out.println(line + "\n");
+        }
+    }
 
     public static void done(String item) {
         int i = Integer.parseInt(item)-1;
```

Comment:
since the "i" here means something, maybe it can be named as "index" or something more meaningful